### DISCUSSION TITLE
others

### DISCUSSION THREAD

**Discussion by Boomz123** [breezy-weather/breezy-weather#940] [Q&A] [answered]
Can you add a feature that automatically reads and copies the SMS verification code

**Comment by papjul** [comment]
What are you talking about?

**Reply by Boomz123** [reply]
on the settings page,in the wights&live wallpaper page,the 'Clock + Daily ( Vertical style ) 'page cannot run

**Reply by papjul** [reply, marked as answer]
Please fill in an issue with the Bug template. Attach your debug log from Settings > Debug

**Reply by Boomz123** [reply]
[5de56915-9868-4d98-913e-81c04ab13eaa.txt](https://github.com/breezy-weather/breezy-weather/files/14974429/5de56915-9868-4d98-913e-81c04ab13eaa.txt)

**Reply by papjul** [reply]
Will be fixed in v5.2.0.
In the meantime, don't use the analog clock, or update to Android >= 12.

**Reply by Boomz123** [reply]
thanks

**Comment by Boomz123** [comment]
It is actually raining, but the card indicates that there will be no rain for the next 120 minutes.
![Screenshot_20240420_193217](https://github.com/breezy-weather/breezy-weather/assets/166996243/ce54b968-a7dc-4221-9438-59f0ba61ef7f)

**Reply by papjul** [reply]
One is current, the other is the MinuteCast. AccuWeather is responsible for the discrepancy between the two.
